### ISSUE TITLE
Cannot seem to get breakpoints working even though sourcemap seems to resolve fine

### ISSUE THREAD

**Issue by jonsvez** [Microsoft/vscode-chrome-debug#343]
launch.json -> 

> 
    "version": "0.1.0",
    "configurations": [
        {
            "name": "Zone",
            "type": "chrome",
            "request": "launch",
            "url": "http://localhost:1465/",
            "diagnosticLogging": true,
            "sourceMaps": true,
            "webRoot": "${workspaceRoot}",
            "sourceMapPathOverrides": {

                "webpack:///./~/*": "${webRoot}/../../node_modules/*",
                "webpack:///./*":   "${webRoot}/*"
            }
            
        }
    ]

----------------------------

Some stuff from log ..

Start -> 
```
Verbose logs are written to C:\Users\jonsve\.vscode\extensions\msjsdiag.debugger-for-chrome-2.4.2\vscode-chrome-debug.txt
2:01:07 PM, 1/29/2017
OS: win32 ia32
Adapter node: v6.5.0 ia32
vscode-chrome-debug-core: 3.8.3
debugger-for-chrome: 2.4.1
spawn('C:\Program Files (x86)\Google\Chrome\Application\chrome.exe', ["--remote-debugging-port=9222","--no-first-run","--no-default-browser-check","http://localhost:1465/"])
Discovering targets via http://127.0.0.1:9222/json
HTTP GET failed: Error: connect ECONNREFUSED 127.0.0.1:9222
Discovering targets via http://127.0.0.1:9222/json
Target userAgent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
Paths.scriptParsed: resolved http://localhost:1465/ReactAppSrc/dist/build.js to c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist\build.js. webRoot: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc
SourceMaps.getMapForGeneratedPath: Finding SourceMap for c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist\build.js by URI: build.js.map and webRoot: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc
SourceMaps.loadSourceMapContents: Reading local sourcemap file from c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist\build.js.map
SourceMap: creating for c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist\build.js
SourceMap: sourceRoot: 
SourceMap: sources: ["webpack:///webpack/bootstrap 7ecd448f0a40a271acca","webpack:///./~/babel-polyfill/lib/index.js","webpack:///./~/babel-polyfill/~/core-js/shim.js","webpack:///./~/babel-polyfill/~/core-js/modules/es6.symbol.js","webpack:///./~/babel-polyfill/~/core-js/modules/_global.js","webpack:///./~/babel-polyfill/~/core-js/modules/_has.js","webpack:///./~/babel-polyfill/~/core-js/modules/_descriptors.js","webpack:///./~/babel-polyfill/~/core-js/modules/_fails.js","webpack:///./~/babel-polyfill/~/core-js/modules/_export.js","webpack:///./~/babel-polyfill/~/core-js/modules/_core.js","webpack:///./~/babel-polyfill/~/core-js/modules/_hide.js","webpack:///./~/babel-polyfill/~/core-js/modules/_object-dp.js","webpack:///./~/babel-polyfill/~/core-js/modules/_an-object.js","webpack:///./~/babel-polyfill/~/core-js/modules/_is-object.js","webpack:///./~/babel-polyfill/~/core-js/modules/_ie8-dom-define.js","webpack:///./~/babel-polyfill/~/core-js/modules/_dom-create.js","webpack:///./~/babel-polyfill/~/core-js/modules/_to-primitive.js","webpack:///./~/babel-polyfill/~/core-js/modules/_property-desc.js","webpack:///./~/babel-polyfill/~/core-js/modules/_redefine.js","webpack:///./~/babel-polyfill/~/core-js/modules/_uid.js","webpack:///./~/babel-polyfill/~/core-js/modules/_ctx.js","webpack:///./~/babel-polyfill/~/core-js/modules/_a-function.js","webpack:///./~/babel-polyfill/~/core-js/modules/_meta.js","webpack:///./~/babel-polyfill/~/core-js/modules/_shared.js","webpack:///./~/bab[...]
SourceMap: webRoot: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc
SourceMap: no sourceRoot specified, using script dirname: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist
SourceMap: mapping webpack:///./~/babel-polyfill/lib/index.js => c:\Users\jonsve\ZitiusProjects\Zone\node_modules\babel-polyfill\lib\index.js, via sourceMapPathOverrides entry - "webpack:///./~/*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/../../node_modules/*"
```

---------------------------------

Then ending -> 
```
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/site.css?a3d8 => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\site.css?a3d8, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/site.css => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\site.css, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/print.css?1db4 => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\print.css?1db4, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/print.css => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\print.css, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/Datetime.css?4909 => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\Datetime.css?4909, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMap: mapping webpack:///./ZitiusApp/ReactAppSrc/css/Datetime.css => c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\Datetime.css, via sourceMapPathOverrides entry - "webpack:///./*": "c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc/*"
SourceMaps.scriptParsed: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\dist\build.js was just loaded and has mapped sources: ["c:\\Users\\jonsve\\ZitiusProjects\\Zone\\ZitiusApp\\ReactAppSrc\\dist\\webpack:\\webpack\\bootstrap 7ecd448f0a40a271acca","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\lib\\index.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\shim.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\es6.symbol.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_global.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_has.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_descriptors.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_fails.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_export.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_core.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_hide.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_object-dp.js","c:\\Users\\jonsve\\ZitiusProjects\\Zone\\node_modules\\babel-polyfill\\~\\core-js\\modules\\_an-object.js","c:[...]
Paths.scriptParsed: could not resolve http://localhost:50276/a6955643e11f4533acb944e4e7583cea/browserLink to a file under webRoot: c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc. It may be external or served directly from the server's memory (and that's OK).
Terminated: Got disconnect request
```

> 
Basically every breakpoint I create gets this error -> "Breakpoint ignored because generated code could not be found etc.."

**Comment by roblourens** [comment]
It looks like your override mapping may not be right -

`c:\Users\jonsve\ZitiusProjects\Zone\ZitiusApp\ReactAppSrc\ZitiusApp\ReactAppSrc\css\site.css?a3d8`

Notice that `ZitiusApp\ReactAppSrc` appears twice.

Try playing with that setting and using the `.scripts` command to check whether it's correct.

**Comment by jonsvez** [comment]
I'm so dumb, thanks alot man <3 !

-> "webpack:///./*":   "${webRoot}/../../*" fixed it !

**Comment by roblourens** [comment]
Glad you got it to work!
